backend/routes/announcements.js

```diff
@@ -1,49 +1,60 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
-const { Announcements} = require('../database');
+const { Announcements } = require("../database");
 
 // Routes
-
+//todo:create an update for announcements
 router.use((request, response, next) => {
   response.header("Access-Control-Allow-Origin", "*");
-  response.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+  response.header(
+    "Access-Control-Allow-Headers",
+    "Origin, X-Requested-With, Content-Type, Accept"
+  );
   next();
 });
 
-router.get('/', (request, response, next) => {
-  Announcements.getAll().then( (result) => {
-    response.status(200).json(result);
-  
-  }).catch((error) => {
-    response.status(400).json(error);
-  
-  });
+router.get("/", (request, response, next) => {
+  Announcements.getAll()
+    .then(result => {
+      response.status(200).json(result);
+    })
+    .catch(error => {
+      response.status(400).json(error);
```

Comment:
I know this was my code, but can you change like 22 to return a new error message like the others? I feel the error caught might provide too much information.